keyboards/chimera_ortho_plus/chimera_ortho_plus.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "quantum.h"
+
+#define red_led_off   PORTF |= (1<<5)
+#define red_led_on    PORTF &= ~(1<<5)
+#define blu_led_off   PORTF |= (1<<4)
+#define blu_led_on    PORTF &= ~(1<<4)
+#define grn_led_off   PORTD |= (1<<1)
+#define grn_led_on    PORTD &= ~(1<<1)
```

Comment:
This block *could* use the [GPIO Control functions](https://docs.qmk.fm/#/internals_gpio_control?id=gpio-control-idgpio-control), but I'm not gonna call it required.